src/components/generic/UserIcon.vue

```diff
@@ -14,10 +17,46 @@ const imageStyle = computed(
       objectFit: 'contain',
       borderRadius: `${props.size || 0}px`,
       display: 'block',
+      cursor: props.idTooltip ? 'pointer' : 'default',
     }) as const,
 )
+
+const userId = computed(() => props.id || user.value?.id)
+const tooltipText = computed(() => `@${userId.value}`)
+
+const handleClick = () => {
+  if (props.idTooltip) {
+    showTooltip.value = true
+  }
+}
+
+// tooltipProps からマウスイベントを除外
+const getModifiedTooltipProps = (tooltipProps: Record<string, unknown>) => {
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const { onMouseenter, ...rest } = tooltipProps // ホバーでは表示させない
+  return rest
+}
 </script>
 
 <template>
-  <img :style="imageStyle" :src="`https://q.trap.jp/api/v3/public/icon/${id || user?.id}`" />
+  <v-tooltip v-if="idTooltip" v-model="showTooltip" location="top">
```

Comment:
```suggestion
  <v-tooltip v-if="idTooltip" v-model="showTooltip" location="top" :open-on-hover="false">
```

getModifiedTooltipsPropsを用いずこういう風にもできそうです。

Reply:
あ、そんな書き方があったんだ（リファレンスよく見てなかった…）
と思ってそっちの実装に切り替えようとしてみたんですが、
:open-on-hover を False にすることで onMouseleave 時の非表示設定も消えてしまうみたいで、もう一回クリックしない限りずっとその Tooltip が表示されたままになります。これは見づらいのでやめておきたいです
https://github.com/traPtitech/rucQ-UI/pull/88/commits/d37ee96c9adf0f818d0d52056e799a1911fd1dc4 で意図をわかりやすくしたり open-on-click だけ採用したりしてみました